server/src/main/java/org/apache/druid/server/lookup/cache/LookupLoadingSpec.java

```diff
@@ -80,6 +88,34 @@ public ImmutableSet<String> getLookupsToLoad()
     return lookupsToLoad;
   }
 
+  public static LookupLoadingSpec createFromContext(Map<String, Object> context, LookupLoadingSpec defaultSpec)
```

Comment:
@cryptoe IIUC, the suggestion is to have a `Map<String, Object> defaultContext` as a class variable in LookupLoadingSpec class?

Then the clients would have to update defaultContext in most cases, which seems unnecessary.

Another option could be to add another method:
```java
public static LookupLoadingSpec createFromContext(Map<String, Object> context) {
    return createFromContext(context, LookupLoadingSpec.ALL);
}
```

Thoughts?

Reply:
I think it is okay to pass the `defaultSpec` to this method, because different call sites may want different defaults if not overridden in the context.

It is similar to `QueryContexts.parseAsString(context, key, defaultValue)`.